docs/developer/accessibility.md

```diff
@@ -0,0 +1,25 @@
+# Accessibility
+
+FAST-DNA has worked diligently from the ground up to enable accessibility in collaboration with legal, accessibility, and marketing teams at Microsoft. As an ecosystem to accelerate the building of Web sites and apps we ensure you have the tools to build accessible user exerpiences.
+
+To learn more about Microsoft's commitment visit their [accessibility](https://www.microsoft.com/en-us/accessibility) website.
+
+Compliments of [@rachelnabors](https://github.com/rachelnabors)
+
+## Browsers
+
+The base component packages support IE11+ and all modern browsers.
+
+### Reporting browser bugs
+
+* [Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/)
+* [Firefox](https://bugzilla.mozilla.org/)
+* [Safari](https://bugs.webkit.org/)
+* [Chrome](https://bugs.chromium.org/p/chromium/issues/list)
+* [Opera](https://bugs.opera.com/wizard/desktop)
+
+## Animation accessibility
+
+* End users should have a mechanism to reduce or remove animations from their experience. The Safari devs have proposed a media query for reduced animation. ("prefers-reduced-motion")
+* Some users with vestibular disorders or other cognitive disabilities have problems with movement. A safe fallback for nearly all users is fade animations.
+* Flashing animations can cause seizures and should be avoided. Good data on the accessibility of animation is scarce.
```

Comment:
I think there should be other sections added, maybe in a separate task, talking about the component unhandled props and other general patterns we are implementing to enable accessibility.

Reply:
Sure, feel free to write a task for new content you'd like to see in here.